app/scripts/models/account.js

```diff
@@ -335,6 +341,10 @@ define(function (require, exports, module) {
       });
     },
 
+    signInReason: function (key) {
+      return this._fxaClient.SIGNIN_REASON[key];
```

Comment:
The need for this function causes me to ask whether we should just extract SIGNIN_REASON into its own module, similar to [VerificationReasons](https://github.com/mozilla/fxa-content-server/blob/signin-reverification/app/scripts/lib/verification-reasons.js) in the verify signin PR.